packages/styles/icon-button.css

```diff
@@ -1,42 +1,42 @@
 .IconButton {
+  position: relative;
   display: inline-flex;
   justify-content: center;
   align-content: center;
   height: var(--button-thin-height);
   width: var(--button-thin-height);
-  border: 1px solid transparent;
   border-radius: 3px;
+  border: none;
   background-color: var(--icon-button-background-color);
   color: var(--icon-button-icon-color);
   margin: 2px;
+  padding: 0;
 }
 
-.IconButton svg {
-  height: calc(var(--button-thin-height) - 4px);
-  width: calc(var(--button-thin-height) - 4px);
+.IconButton div {
```

Comment:
One more small thing I noticed, do you think this should be more specific? i.e. `.IconButton .Icon`? 

`div` seems a bit too generic.

Reply:
Fixed